### PR TITLE
PROGRAMMES-5852 Fix param utcoffset repeated in browser

### PR DESCRIPTION
**Pull request by tatitati**
JIRA:
https://jira.dev.bbc.co.uk/browse/PROGRAMMES-5885

### CODE REVIEW

**Comment by felipeparaujo** [comment]
Should you be refactoring rewriteUrlWithUtcOffset instead of removing the attribute?

**Comment by BjornTwachtmann** [comment]
@felipeparaujo that was my suggestion. Simply modify the search parameters instead of looking for a string 50 times. I can't see any downside to removing the attribute once the link is processed, I could be wrong though.

File: app/Resources/js/timezone-notification.js [hunk 1]
```diff
@@ -50,6 +50,7 @@ define('timezone-notification', ['jquery-1.9', 'rv-bootstrap'], function ($) {
                     'Sat'
                 ]
             },
+            attrTargetAddUtcOffset: 'data-href-add-utcoffset',
```
Comment (votemike) [inline app/Resources/js/timezone-notification.js]: This is already specified inside cssSelectors isn't it? Can't you access it from there?

Comment (tatitati) [reply app/Resources/js/timezone-notification.js]: no because of the square brackets in that selector.

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

**Pull request merged**